### ISSUE TITLE
[iOS] 'Castle/Castle.h' file not found

### ISSUE THREAD

**Issue by dericksa** [castle/castle-react-native#22]
Hello,

When building with the 1.0.10 version of Castle pod, we are receiving this  ''Castle/Castle.h' file not found' on RCTCastle.m.
A workaround for this was to change the Castle pod version to 1.0.9, and then it works.

Is there some sort of incompatibility with 1.0.5 from castle-react-native and 1.0.10 from castle-ios?

**Comment by aalekz** [comment]
Hi @dericksa!

Thanks for reporting the issue, we haven't been explicitly setting a specific version in the podspec which resulted in this issue (cocoapods using version 1.0.10 which changed the file structure of the project). I've opened a pull request for this here: https://github.com/castle/castle-react-native/pull/23.

As soon as we have the update in place for Android as well we'll publish a new release of the library. I hope that your work-around will suffice in the meantime.

I'll update you once we've released the new version.
